kube/terraform/examples/gcp/main.tf

```diff
@@ -0,0 +1,44 @@
+provider "google" {
+  project     = "${var.gcp_project_id}"
+  region      = "${var.gcp_region}"
+}
+
+resource "google_container_cluster" "primary" {
+  name               = "m3db-bundle-cluster"
+  location           = "${var.gcp_region}"
+  initial_node_count = 3
+
+  
+  # Setting an empty username and password explicitly disables basic auth
+  master_auth {
+    username = ""
+    password = ""
+  }
+
+  node_config {
+    preemptible  = false
+    machine_type = "n1-standard-16"
```

Comment:
Is there any way to make this a user-overrideable variable?

Reply:
Yeah will do.